Legion.BackEnd/tests/Unit/EventTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tests\Unit;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
+use App\User;
+use App\Event;
+use Carbon\Carbon;
+
+class EventTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /** @test */
+    public function it_creates_an_announcement_by_a_user()
```

Comment:
Function names weren't updated after copypasta :P

Reply:
Oh rip. Let me get that.

Reply:
No worries, I know how that goes, believe me xD

Reply:
Fixed